### PR TITLE
#712 Align GGraph model with newest changes from glsp-server

### PR DESCRIPTION
**Pull request by ndoschek**
- Align model in packages/graph with GGraph/SGraph
  - Add missing interface GHtmlRoot
  - Add missing static builder function for GIssueMarker
  - Rename files to match class/interface name
- Update build scripts and VSCode build task
- Update CHANGELOG

Resolves https://github.com/eclipse-glsp/glsp/issues/712

Contributed on behalf of STMicroelectronics.

### CODE REVIEW

File: packages/graph/src/gshape-prerendered-element.ts [hunk 1]
```diff
@@ -27,16 +26,14 @@ import { GPreRenderedElement } from './gpre-rendered-element';
...
-export class GShapePreRenderedElement extends GPreRenderedElement implements GBoundsAware, GAlignable {
+export class GShapePreRenderedElement extends GPreRenderedElement implements GBoundsAware {
     static override builder(): GShapePreRenderedElementBuilder {
         return new GShapePreRenderedElementBuilder(GShapePreRenderedElement);
     }
 
+    position?: Point;
+    size?: Dimension;
     [GBoundsAware] = true;
-    [GAlignable] = true;
```
Comment (tortmayr) [inline packages/graph/src/gshape-prerendered-element.ts:36]: ShaprePreRenderedElements are alignable in sprotty (https://github.com/eclipse/sprotty/blob/139fce21baab249b3ec9bc9e6d519558b876a8f9/packages/sprotty/src/lib/model.ts#L110) so we should also implement the interface

**Comment by ndoschek** [comment]
Thanks @tortmayr for your review, I updated my commit, would be great if you could have another look thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Eclipse GLSP Server Node Changelog
 
+## v1.1.0 - upcoming
+
+### Changes
+
+-   [graph] Align GGraph model with newest changes from glsp-server [#22](https://github.com/eclipse-glsp/glsp-server-node/pull/22) - Contributed on behalf of STMicroelectronics
```
Comment (tortmayr) [inline CHANGELOG.md:7]: Please move this in the breaking changes section as Some gmodel classes/interfaces have been renamed.
Maybe also shortly note down the classes in question (e.g. `GLayoutContainer` (and related classes) => `GLayouting`))

**Review by tortmayr — APPROVED** [review]
Thanks Nina. Code looks good to me and everything still works as expected.
I only have one small remark regarding the changelog

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+### Breaking Changes
+
+-   [graph] Align GGraph model with newest changes from glsp-server [#22](https://github.com/eclipse-glsp/glsp-server-node/pull/22) - Contributed on behalf of STMicroelectronics
+    -   Renamed interfaces:
+        -   `EdgePlacement` -> `GEdgePlacement` (affected classes: `GEdgeLayoutable`, `GLabel`)
+        -   `GLayoutContainer` (`glayout-container.ts`) -> `GLayouting` (`glayouting.ts`) (affected classes: `GCompartment`, `GGraph`, `GNode`)
+        -   `GShapePreRenderedElement` (`gpre-shape-prerendered-element.ts`) -> `GShapedPreRenderedElement` (`gshaped-prerendered-element.ts`)
+    -   Renamed file names: `gbound-aware.ts` -> `gbounds-aware.ts`
+    -   `GGraph` now additionally extends `GLayoutable`
```
Comment (tortmayr) [inline CHANGELOG.md:13]: API additions don' need to be mentioned in the breaking change section (as they don't break any existing API :wink:)
Please remove the last three lines

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
 # Eclipse GLSP Server Node Changelog
 
+## v1.1.0 - upcoming
+
+### Breaking Changes
+
+-   [graph] Align GGraph model with newest changes from glsp-server [#22](https://github.com/eclipse-glsp/glsp-server-node/pull/22) - Contributed on behalf of STMicroelectronics
+    -   Renamed interfaces:
+        -   `EdgePlacement` -> `GEdgePlacement` (affected classes: `GEdgeLayoutable`, `GLabel`)
+        -   `GLayoutContainer` (`glayout-container.ts`) -> `GLayouting` (`glayouting.ts`) (affected classes: `GCompartment`, `GGraph`, `GNode`)
```
Comment (tortmayr) [inline CHANGELOG.md:10]: You can omit the file information here ('glayout-container.ts`) etc. GLayoutContainer is default exported so a source file name changes does not break anything for adopters.

**Pull request merged**